src/signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/styled.tsx

```diff
@@ -97,7 +97,7 @@ export const AddressPanel = styled.article`
   z-index: 2;
   left: 0;
   top: 0;
-  animation: ${slideUp} 0.3s cubic-bezier(0.4, 0, 0.2, 1)
+  animation: ${slideUp} 0.3s cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Should the semi-colon be added here? Look like translate3d belongs as argument to animation right?